megamek/src/megamek/client/ui/swing/ForceGenerationOptionsPanel.java

```diff
@@ -389,20 +373,22 @@ public void updateSubfactionChoice() {
     }
 
     /**
-     * When faction or subfaction is changed, refresh ratings combo box with appropriate
+     * When faction or sub-faction is changed, refresh ratings combo box with appropriate
```

Comment:
Unnecessary hyphen. Subfaction is perfectly valid.

Reply:
Reverted, although technically subfaction is not a word.

Reply:
Language is defined by use. It is an element of language that conveys meaning, and is therefore a word.